packages/gatsby-theme-adr/src/components/SEO.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren } from 'react';
+import { useSiteMetadata } from '../hooks/use-site-metadata';
+
+export type SEOProps = PropsWithChildren<{
+  title?: string;
+  description?: string;
+  pathname?: string;
+}>;
+
+const SEO = ({ title, description, pathname, children }: SEOProps) => {
```

Comment:
Does SEO have the same purpose than Helmet? 
Shouldn't be better to name this with Helmet? 
Every time that I see "Helmet" in a React application I think about `<title>`, `<meta>` and so forth.

For me, if I'd have this ADR for internal purposes SEO doesn't makes sense. the `<meta name="twitter...">` are not going to be used, but we have here the title, description and image, and can be used.

Reply:
> For me, if I'd have this ADR for internal purposes SEO doesn't makes sense.

I think ADRs are not internal only. We want them promoted and discoverable.